src/3_PB/AnalisiDeiRequisiti/content.typ

```diff
@@ -90,55 +89,37 @@ Relativamente all'utilizzo della #glossary("dashboard"), viene definito un unico
   caption: [UC0 Visualizzazione menù #glossary("dashboard")]
 )
 
-/*TODO: ricontrollare la fine di ogni use case che comprende l'uso di un #glossary("pannello")*/
-=== UC1: Visualizzazione dashboard generale sensori
+=== UC1: Visualizzazione #glossary[dashboard] dati grezzi
 - *Attore Principale*: #glossary("amministratore pubblico")\;
 - *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
-- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza #glossary("pannelli") relativi allo stato dei sensori, all'interno di un'unica #glossary[dashboard]\;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente i dati grezzi inviati dai sensori, all'interno di un'unica #glossary[dashboard]\;
 - *Scenario Principale*:
   + L'#glossary("amministratore pubblico") accede alla piattaforma di visualizzazione; 
-  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary("dashboard") generale relativa ai sensori.
+  + L'#glossary("amministratore pubblico") seleziona la visualizzazione della #glossary("dashboard") relativa ai dati grezzi.
 
 #figure(
-  image("assets/UML/UC1_Visualizzazione-dashboard-generale-sensori.png",width:70%),
-  caption: [UC1 Visualizzazione #glossary("dashboard") generale sensori]
+  image("assets/UML/UC1_Visualizzazione-dashboard-dati-grezzi.png",width:70%),
+  caption: [UC1 Visualizzazione #glossary("dashboard") dati grezzi]
 )
 
-
-#pagebreak()
-=== UC1.1: Visualizzazione posizione sensori su mappa
+=== UC1.1: Visualizzazione #glossary("pannello") dati grezzi
 - *Attore Principale*: #glossary("amministratore pubblico")\;
-- *Precondizioni*: il #glossary("sistema") è operativo e accessibile;
-- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza un #glossary("pannello") contenente una mappa che mostra le posizioni dei sensori, visualizzati tramite icone, su di essa. Le icone dei sensori sono colorate in base al tipo di #glossary("sensore")  e riportano una label che ne esplicita il tipo, il nome e le coordinate geografiche; 
+- *Precondizioni*: l'#glossary("amministratore pubblico") ha selezionato la visualizzazione relativa al dominio dei dati grezzi;
+- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza il #glossary("pannello") relativo ai dati grezzi, che riporta il nome del #glossary[sensore], la tipologia del #glossary[sensore], il timestamp della rilevazione e il valore della misurazione, in forma tabellare, per ogni tipo di #glossary[sensore]; la tabella contenente i dati menzionati è indicizzata tramite il nome dei sensori e, nel caso in cui la misurazione sia composta da più dati, tutti i valori sono elencati ed etichettati opportunamente all'interno della stessa entrata nella colonna corrispondente;
```

Comment:
siamo sicuri che la tabella sia effettivamente indicizzata sulla base del nome del sensore, non dovrebbe voler dire che è c'è una entry per ogni nome di sensore? 
```suggestion
- *Postcondizioni*: l'#glossary("amministratore pubblico") visualizza il #glossary("pannello") relativo ai dati grezzi, che riporta il nome del #glossary[sensore], la tipologia del #glossary[sensore], il timestamp della rilevazione e il valore della misurazione, in forma tabellare, per ogni tipo di #glossary[sensore]; la tabella contenente i dati menzionati è indicizzata tramite il nome dei sensori e, nel caso in cui la misurazione sia composta da più dati, essi vengono presentati all'interno della stessa cella corrispondente alla colonna della misurazione;
```

Reply:
in riferminento a questa cosa dell'indicizzazione, non credo voglia dire niente. forse sarebbe da togliere completamente di mezzo 


Quando si dice che una tabella è indicizzata, si intende che è stata creata un'indicizzazione su una o più colonne di quella tabella all'interno di un database. Questo significa che il database ha creato una struttura di dati aggiuntiva che ordina e organizza i valori delle colonne specificate per consentire un accesso più rapido ai dati. In breve, l'indicizzazione di una tabella rende più efficienti le operazioni di ricerca, filtraggio e ordinamento dei dati all'interno di quella tabella, poiché il database può utilizzare gli indici per accedere direttamente ai dati desiderati anziché scansionare l'intera tabella. Questo può portare a notevoli miglioramenti delle prestazioni, specialmente su grandi set di dati.

Reply:
sarei propenso a toglierla